assets/app/scripts/services/logLinks.js

```diff
@@ -50,19 +50,42 @@ angular.module('openshiftConsole')
         newTab(params);
       };
 
-      // Not currently used.
-      // new tab: path/to/current?view=textonly
-      var textOnlyLink = function() {
-        newTab({view: 'textonly'});
+      // Kibana archive URL v1: using a string & _.template
+      var archiveTpl = [
+        '<%= baseurl %>',
+        '/#/discover?',
+        '_g=',
+        '(refreshInterval:(display:Off,pause:!f,section:0,value:0),',
+        'time:(from:now-7y,mode:quick,to:now))&_a=(columns:!(_source),',
+        'filters:!((meta:(disabled:!f,index:\'<%= namespace %>.*\',key:kubernetes_namespace_name,negate:!f,value:<%= namespace %>),',
+        'query:(match:(kubernetes_namespace_name:(query:<%= namespace %>,type:phrase)))),',
+        '(meta:(disabled:!f,index:\'<%= namespace %>.*\',key:kubernetes_pod_name,negate:!f,value:<%= podname %>),',
+        'query:(match:(kubernetes_pod_name:(query:<%= podname %>,type:phrase))))),',
+        'index:\'<%= namespace %>.*\',interval:auto,query:\'\',sort:!(time,desc))'
+      ].join('');
```

Comment:
yeah... i kind of preferred the `<% var %>` since this is a pretty gnarly url with all kinds of syntax, made it really obvious what was a variable, but I think I got it whittled down to something reasonable.  Just about to push an update.